.github/workflows/dev-docker-image-build.yml

```
@@ -17,8 +17,12 @@ jobs:
           with:
             ref: ${{ github.ref.name }}
         - uses: ./.github/actions/branch-build-and-push
-          env:
-            DOCKER_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+          with:
+            ghcr-password: ${{ secrets.GITHUB_TOKEN }}
+            dockerhub-password: ${{ secrets.DOCKERHUB_TOKEN }}
+            push-to-ghcr: true
+            # Only push 'dev' branch to Docker Hub
+            push-to-dockerhub: ${{ github.ref.name == 'refs/heads/dev' }}
```

actionlint (1.7.7)

receiver of object dereference "name" must be type of object but got "string"

(expression)